ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandStore.java

```diff
@@ -338,6 +328,11 @@ private long getCommandTopicOffset() {
     }
   }
 
+  @Override
+  public boolean corruptionDetected() {
```

Comment:
Suggestion (up to you to change it):

Shouldn't we use a different term instead of corruption? For me, corruption means something happened in the backup file that cannot be read or deserialized (perhaps due to a file or line corrupted, missing byte or something). But in this case it's just the command topic is missing or the command topic is out of sync with the backup file. Perhaps `syncErrorDetected` ?

Same comment about the `commandTopicBackup.commandTopicCorruption()`